src/content/docs/new-relic-solutions/observability-maturity/operational-efficiency/observability-coe.mdx

```diff
@@ -28,54 +28,53 @@ import omaCoeChannels from 'images/oma_diagram_coe-channels.png'
 import omaCoeKnowledgeBase from 'images/oma_diagram_coe-knowledge-base.png'
 
 
-## Overview [#overview]
-### What is a CoE? [#coe-definition]
+## What is a CoE? [#coe-definition]
 
-By definition a center of excellence is… 
+By definition, a center of excellence is… 
 
-_A team of dedicated individuals, from multiple functional areas of an organization, that lead the way in exploring and adopting tools, techniques and practices._ 
+* A team of dedicated individuals, from multiple functional areas of an organization, that lead the way in exploring and adopting tools, techniques and practices.
 
-_The Center of Excellence promotes collaboration, standards, best practice and adoption. It is supported by a community with a rich collaborative knowledge-base reinforced with frequent enablement and learning._
+* The Center of Excellence promotes collaboration, standards, best practice and adoption. It is supported by a community with a rich collaborative knowledge-base reinforced with frequent enablement and learning.
 
-Centers of excellence exist around a host of disciplines including things like security, applications, cloud services and automation. This guide is focused on how to create a Center of Excellence around observability best practices. This will help you drive standards, speed and scale across your organization, and maximize the return on the investment in observability solutions.
+CoE's exist around a host of disciplines including things like security, applications, cloud services and automation. This guide focuses on how to create a CoE around observability best practices. This will help you drive standards, speed and scale across your organization, and maximize the return on the investment in observability solutions.
```

Comment:
```suggestion
CoEs exist around a host of disciplines including things like security, applications, cloud services and automation. This guide focuses on how to create a CoE around observability best practices. This will help you drive standards, speed and scale across your organization, and maximize the return on the investment in observability solutions.
```